src/test/java/com/greenfiling/smclient/AccountClient_Manual.java

```diff
@@ -18,19 +18,26 @@
 
 import org.junit.BeforeClass;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.greenfiling.smclient.internal.JsonHandle;
 import com.greenfiling.smclient.model.Account;
 import com.greenfiling.smclient.model.Links;
 import com.greenfiling.smclient.model.exchange.Show;
 
 public class AccountClient_Manual {
+  @SuppressWarnings("unused")
+  private static final Logger logger = LoggerFactory.getLogger(AccountClient_Manual.class);
+
```

Comment:
I see this change done in all test classes even where the logger is not used. Just curious if this is done to enforce a standard for future tests that might need it.

Reply:
Yeah, just a judgement call.  My thought process is something like:

* Don't include the logger if we're not using it
* but if we do need it, it's not there, adding friction to using it.  When that happens we end up with bad decisions like logE
* Maybe if we're not using it we could comment it out.  My problem with that is that it will strip the imports, and since some classes use slf4j and some use the core java logging, it can be a pain to remember which imports are correct.
* So, have it exist and be available all the time so a person who needs it doesn't have to think about using it

There's still one issue with this, which is the SuppressWarnings annotation.  If I don't have it, I get a warning on the unused var.  If I do add it but later use the variable, I get an INFO about using the annotation when it doesn't get used. I think my solution is to comment/uncomment the annotation as needed, but never remove it fully.

There's a very real chance I've thought about this too much.  Going to leave them instantiated for the purposes of getting this committed, but open to discussion in the future